recipe/meta.yaml

```diff
@@ -61,6 +61,9 @@ about:
   dev_url: https://github.com/numba/llvmlite
 
 extra:
+  skip-lints:
+    - should_use_compilers
+    - potentially_bad_ignore_run_exports
```

Comment:
It could be nice to add a comment to explain why the ignore_run_exports is necessary.

Reply:
I am removing zlib actually, it doesn't seem to be needed.
I added a comment for should_use_compilers.

Reply:
Thank you!